Remove last vestiges of Raphael

mistergone opened the pull request:
Deletes raphael-min.js from local_static directory

## Testing
- Everything should work just as it did before.

## Review
- @chosak @schbetsy 

## Screenshots
<img width="920" alt="screen shot 2018-09-19 at 4 32 31 pm" src="https://user-images.githubusercontent.com/1490703/45779671-a869ff00-bc29-11e8-883a-fbbb702f968d.png">

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

mistergone committed:
Remove last vestiges of Raphael
- Delete raphael-min.js from local_static directory

chosak commented:
I see two other references to this file in this project:

1. This [SpecRunner-sauce.html](https://github.com/cfpb/college-costs/blob/master/paying_for_college/local_static/paying_for_college/disclosures/SpecRunner-sauce.html#L26) file. I'm not sure the purpose of this file: is it obsolete?
2. This [worksheet-old.html](https://github.com/cfpb/college-costs/blob/master/paying_for_college/templates/worksheet-old.html#L79) file, which was [archived back in 2006](https://github.com/cfpb/college-costs/commit/29f085920d991cc14a955d43a498ed1d26376637#diff-d733d2840f49215d8fd1adfcdacd1725). Can this file also be removed?

mistergone committed:
Removed old files

- Removed two static HTML files which are no longer in use (and contain references
to raphael-min.js)

chosak reviewed (approved):
![](https://i.ytimg.com/vi/PihHfXaYevo/hqdefault.jpg)
